### PR TITLE
docs: setup nextra with react-live, add draft content

### PR DESCRIPTION
**Pull request by francisco-guilherme**
- setup `nextra` with `unocss` and `react-live`
- add docs draft content and home page
- fix issues with `SSR` compatibility

### CODE REVIEW

File: apps/docs/uno.config.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { defineConfig, PresetOrFactory, UserConfig } from "unocss";
+import { presetHv } from "@hitachivantara/uikit-uno-preset";
+
+const config: UserConfig = {
+  presets: [presetHv() as PresetOrFactory<object>],
+};
+
+export default defineConfig(config);
```
Comment (zettca) [inline apps/docs/uno.config.ts:8]: `UserConfig` is widening the types unnecessarily. If we just inline it, the type issue goes away

```suggestion
export default defineConfig({
  presets: [presetHv() as PresetOrFactory<object>],
});
```

File: packages/uno-preset/src/theme.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const extendTheme: ThemeExtender<Theme> = (baseTheme) => ({
 
   // colors
   colors: {
+    ...baseTheme.colors,
```
Comment (zettca) [inline packages/uno-preset/src/theme.ts:39]: Nice 👍 

We don't need any of the colors below now, they're [included in the `baseTheme`](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/_theme/colors.ts#L10-L14)

**Review by zettca — APPROVED** [review]
Looking great 👍 

With this, I guess we should re-enable the nextra deployments [in the documentation workflow](https://github.com/lumada-design/hv-uikit-react/blob/master/.github/workflows/documentation.yml#L16-L17), at least for the `master` branch

Some small notes and comments:

File: apps/docs/src/components/home/Home.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    position: "absolute",
+    width: "100%",
+    height: "100%",
+    background: "transparent",
+    backgroundImage:
+      "linear-gradient(#f4f4f4 0.1em, transparent 0.1em), linear-gradient(90deg, #f4f4f4 0.1em, transparent 0.1em)",
+    backgroundSize: "4em 4em",
+  },
+  mainContainerHeight: {
+    height: "calc(100vh - 64px)",
```
[the remark this answers is not in the document]
Comment (plagoa) [reply apps/docs/src/components/home/Home.tsx:17]: `w` or `h` ?

File: apps/docs/uno.config.ts (hunk 1 above)
Comment (francisco-guilherme) [reply apps/docs/uno.config.ts]: Changed to .mjs for consistency

File: apps/docs/src/components/home/Home.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (francisco-guilherme) [reply apps/docs/src/components/home/Home.tsx:17]: I tried that, but for some reason didn't work.

[the remark this answers is not in the document]
Comment (francisco-guilherme) [reply apps/docs/src/components/home/Home.tsx:17]: stupid me, it worked 🤦

**Pull request merged**